src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj

```diff
@@ -163,27 +164,32 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Microsoft.CSS.Core, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+    <Reference Include="Microsoft.WebTools.Languages.Css">
```

Comment:
Can we mop these up bit to use this pattern instead?

    <Reference Include="..\..\lib\Microsoft.WebTools.Languages.Json.dll" />
      <Private>False</Private> 
    </Reference>

It saves a few lines and makes the reference path explicit (HintPath is used if it isn't resolved by another automatic means, and if that *were* to happen then it's not using the right reference assembly)